### DISCUSSION TITLE
hacktoberfest 2022

### DISCUSSION THREAD

**Discussion by nolith** [magicbug/Cloudlog#1666] [General]
Any thought on joining the hacktoberfest 2022 @magicbug ?

It only needs adding the `hacktoberfest` topic to the project.

https://hacktoberfest.com/participation/#maintainers

**Comment by magicbug** [comment]
Not currently something that I'm looking at partaking in for a T-shirt, we expect PRs all year round :)

**Reply by nolith** [reply]
Fair enough. I usually see those opportunities as a promotional effort to gain new contributors. There is a search system that will give visibility to the project. 

But given the special nature of this project the hamradio crowd partaking in hacktoberfest may be very small indeed.
